File: SEGMENTATION/Biggest_Border.m
function [location]=Biggest_Border(InnerBorder)
    max=0;
    %Send locations of biggest border in array 'InnerBorder'
    for i=1:1:length(InnerBorder)
        if(length(InnerBorder{i})>max)
            max=length(InnerBorder{i});
            location=i;
        end
    end
    %check points
    %disp(max)
end

File: SEGMENTATION/python/Biggest_Border.py
import numpy as np


import numpy as np
def Biggest_Border(InnerBorder=None, *args, **kwargs):
    mx = 0

    for i in np.arange(1, len(InnerBorder), 1).reshape(-1):
        if len(InnerBorder[i]) > mx:
            mx = len(InnerBorder[i])


            location = i

    return location


if __name__ == '__main__':
    pass